torch/_inductor/config.py

```diff
@@ -1044,6 +1044,10 @@ class triton:
     # Whether to upcast float16 / bfloat16 to float32 in triton codegen (Experimental)
     codegen_upcast_to_fp32 = True
 
+    # Whether persistent matmul kernels should be enabled this flag only as effect when on h100
+    # with a verison of triton new enough to support TMA
+    enable_persistent_matmul = os.environ.get("ENABLE_PERSISTENT_MATMUL", "0") == "1"
```

Comment:
Should we make TMA explicit in the config name?

Reply:
Spoke offline, we think it could be valuable to have both a 
persistent_flag and a TMA flag since they are independent and mm may want to use both.

I combined into 1 since for scaled_mm I found the eager version to output perform triton when the features were used independently and persistent + TMA appeared to be universally better in the cases I was looking at. 

Another question is whether this should actually be set to True by default. Since I have decent guarding checks.